jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -252,10 +251,9 @@ public XmlParser getXmlParser()
      * Reads and parses the XML configuration file.
      *
      * @param resource the Resource to the XML configuration
-     * @throws IOException if the configuration could not be read
-     * @throws SAXException if the configuration could not be parsed
+     * @throws XmlConfigurationException if configuration was not able to loaded from XML provided
      */
-    public XmlConfiguration(Resource resource) throws SAXException, IOException
+    public XmlConfiguration(Resource resource)
```

Comment:
Uhm, this is now a breaking change, as user code that did `try/catch (IOException)` will now get a compilation error.

Reply:
Guess I can leave those `throws` declarations to not change the signature, 
Doesn't matter if that they will never happen with the other changes in this PR.